src/HtmlSanitizer/HtmlSanitizer.cs

```diff
@@ -578,6 +578,21 @@ public IHtmlDocument SanitizeDom(string html, string baseUrl = "")
             return dom;
         }
 
+        /// <summary>
+        /// Sanitizes the specified parsed HTML body fragment.
+        /// The Document Must have been pared with CSS support and the following options enabled
+        /// "IsIncludingUnknownDeclarations", "IsIncludingUnknownRules" and "IsToleratingInvalidSelectors"
+        /// </summary>
+        /// <param name="document">The pared HTML Document.</param>
```

Comment:
-> parsed

Reply:
Fixed

Reply:
Still showing "pared".

Reply:
Fixed it in one place not the other 4...